Update Promtail modul to install with package name (rpm/deb)

janfickler opened the pull request:
- Update Promtail modul to install with package name (rpm/deb)
- Update metadata.json - OS version support and modul-supportversions

bastelfreak committed:
allow promtail installation via package

janfickler committed:
Update metadata.json

add as supported OS AlmaLinux 8 and 9 and update other OS-Systems for later versions that are working

janfickler committed:
Merge pull request #1 from janfickler/package

Package

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/puppet-promtail?pullRequest=48) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/grafana/puppet-promtail?pullRequest=48) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: janfickler<br/>:x: bastelfreak<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grafana/puppet-promtail?pullRequest=48) it.</sub>

janfickler committed:
Update init.pp

janfickler committed:
Update init.pp

janfickler committed:
Update init.pp

janfickler committed:
Update config.pp

change config.yaml to config.yml like it is used from the service-file in the rpm

janfickler committed:
Update promtail.service

add log-config for rsyslog to service-file

janfickler committed:
Update service.pp

add support for service file for package installation and archive installation

janfickler committed:
Update config.pp

janfickler committed:
make all initial variables optional

janfickler committed:
config change standard values

janfickler committed:
update config standard

janfickler committed:
update

janfickler committed:
Update init.pp

janfickler committed:
Update init.pp

janfickler committed:
Update metadata.json

janfickler committed:
Update init.pp

janfickler committed:
Update install.pp

janfickler committed:
Update init.pp

janfickler committed:
Update promtail.service

kasimon commented:
Any chance of moving this forward? It would be much simpler now to use the official grafana repository, mainly in order to keep promtail uptodate without changing the puppet code on every update.

genebean commented:
As best as I can tell, Grafana may have abandoned this repo and my permissions won't let me help you out. I plan to be at ConfigMgmtCamp at the beginning of Feb so maybe I can get someone there to help, but otherwise I think we may need to look at forking to Vox Pupuli or similar to keep this going.

janfickler committed:
Update init.pp

janfickler committed:
Update config.pp

janfickler committed:
Update init.pp

janfickler committed:
Update config.pp

janfickler committed:
Update promtail.service
